cep-????.md

```diff
@@ -0,0 +1,160 @@
+# CEP ???? - Virtual packages
+
+<table>
+<tr><td> Title </td><td> Virtual packages </td>
+<tr><td> Status </td><td> Draft </td></tr>
+<tr><td> Author(s) </td><td> Jaime Rodríguez-Guerra &lt;jaime.rogue@gmail.com&gt;</td></tr>
+<tr><td> Created </td><td> Dec 17, 2024</td></tr>
+<tr><td> Updated </td><td> Dec 17, 2024</td></tr>
+<tr><td> Discussion </td><td> https://github.com/conda/ceps/pull/103 </td></tr>
+<tr><td> Implementation </td><td> https://github.com/conda/conda/tree/24.11.1/conda/plugins/virtual_packages, https://github.com/mamba-org/mamba/blob/libmamba-2.0.5/libmamba/src/core/virtual_packages.cpp, https://github.com/conda/rattler/tree/rattler-v0.28.8/crates/rattler_virtual_packages/src </td></tr>
+</table>
+
+> The key words "MUST", "MUST NOT", "REQUIRED", "SHALL", "SHALL NOT", "SHOULD", "SHOULD NOT",
+  "RECOMMENDED", "NOT RECOMMENDED", "MAY", and "OPTIONAL" in this document are to be interpreted as
+  described in [RFC2119][RFC2119] when, and only when, they appear in all capitals, as shown here.
+
+## Abstract
+
+This CEP standardizes which virtual packages MUST be offered by conda solvers.
+
+## Specification
+
+A virtual package is defined as a package record with three fields: name, version and build string.
+The name MUST start with double underscore. The version and build string MUST follow the same semantics as in regular package records.
+
```

Comment:
This seems to be a common feature. So let's refactor it out here and drop from the sections below.

```suggestion

The build string MAY be zero (`0`). Some exceptions apply. See below.

```

Also noticed some caveats. So used the same formula as with the environment variable `CONDA_OVERRIDE_*`